openc3-cosmos-init/plugins/packages/openc3-vue-common/src/tools/base/UserMenu.vue

```diff
@@ -100,19 +135,32 @@ export default {
   data: function () {
     let user = OpenC3Auth.user()
     return {
+      api: new OpenC3Api(),
       showUserMenu: false,
       authenticated: !!localStorage.openc3Token,
       name: user['name'],
       // preferred_username is returned by the token (see authorization.rb)
       username: user['preferred_username'],
       showUpgradeToEnterpriseDialog: false,
       activeUsers: ['None'],
+      newsFeed: false,
+      news: [],
     }
   },
+  computed: {
+    unreadNews: function () {
+      return this.news.filter((news) => !news.read)
+    },
+  },
   watch: {
-    // Whenever we show the user menu, refresh the list of active users
+    // Whenever we show the user menu, read the news and refresh the list of active users
     showUserMenu: function (newValue, oldValue) {
       if (newValue === true) {
+        this.news.forEach((news) => {
+          news.read = true
+        })
+        localStorage.newsRead = this.news.length
```

Comment:
This will break when we start deleting old news.   Maybe use a id / timestamp from the last news item?

Reply:
I guess I could grab the newest timestamp and then look to see if there is anything newer.